src/main/java/uk/gov/register/resources/v2/RecordResource.java

```
@@ -113,8 +101,16 @@ private IndexSizePagination setUpPagination(@QueryParam(IndexSizePagination.INDE
         return pagination;
     }
 
-    private RecordListView getRecordsView(int limit, int offset) throws FieldConversionException {
-        List<Record> records = register.getRecords(EntryType.user, limit, offset);
+    private RecordListView getRecordsView(int limit, int offset, Optional<String> attributeName, Optional<String> attributeValue) throws FieldConversionException {
+        List<Record> records;
+
+        if(attributeName.isPresent() && attributeValue.isPresent()) {
+
+            records = register.getRecordsFacetedByKeyValue(attributeName.get(), attributeValue.get(), limit, offset);
+        }
+        else {
```

Should we return an error if they pass in only one of the two params?